cli/flox/doc/services/flox-services-status.md

```diff
@@ -0,0 +1,63 @@
+---
+title: FLOX-SERVICES-STATUS
+section: 1
+header: "Flox User Manuals"
+...
+
+# NAME
+
+flox-services-status - display the status of services
+
+# SYNOPSIS
+
+```
+flox [<general-options>] services status
+     [ -d=<path> ]
+     [ --json ]
+     [<name>] ...
+```
+
+# DESCRIPTION
+
+Displays the status of one or more services.
+
+If no services are specified, then all services will be displayed. If no
+services have never been started for this environment, an error will be
+displayed. An error will also be displayed if one of the specified services
+does not exist.
+
+Services can have the following statuses:
+- Disabled: this service has never been started
+- Running: this service is currently running
+- Completed: this service has exited
```

Comment:
question nonblocking: I think there are a bunch of other possible states process-compose returns, do we ever pass those through? Full list here https://github.com/F1bonacc1/process-compose/blob/8d6a662c71d24608daf93b51ca1d462a0d5725f9/src/types/process.go#L125-L137

Reply:
It looks like we pass them all through, I thought we did some kind of filtering or collating but I was mistaken. I deleted the sections about the possible statuses because I don't think we need a page-long description of all 8,000 states that `process-compose` lists.